src/components/Period.vue

```diff
@@ -1,11 +1,43 @@
 <template>
-  <div
-    class="period"
-    :class="{ 'not-mobile': !forceMobileLayout, invert }"
-  >
-    <div class="circle" :style="{ fontSize: periodFontSize }">
+  <div class="period" :class="{ 'not-mobile': !forceMobileLayout, invert }">    <svg
+      class="progress"
+      v-if="
+        progress != 0 &&
+        !disableProgressBar &&
+        (actualPeriod.length == 1 || actualPeriod.length == 2)
```

Comment:
If I'm not mistaken, this seems to be enabling the progress bar only for periods with numeric names/half periods (i.e. "1", "1A", etc.). Is there a reason for disabling the progress bar on other periods (such as "Activity"), or am I misunderstanding what this does?

Reply:
yes the periods need to be 1 or 2 letters long because otherwise it cuts off the letters. 
I don't yet have a solution for accommodating longer period names.

<img width="206" alt="Screen Shot 2021-08-11 at 11 23 20 AM" src="https://user-images.githubusercontent.com/31457361/129066484-5dc6bf52-20e0-4a3e-9927-62d7e63ea119.png">

Reply:
here's one idea. For passing periods, and periods where the name of the period is longer (like "activity"), we could put a big progress bar at the top.
<img width="1279" alt="Screen Shot 2021-08-11 at 11 25 51 AM" src="https://user-images.githubusercontent.com/31457361/129066971-50fdaf80-761a-4105-ac16-c0eb4626c445.png">

Reply:
Oh wow I'm being stupid, wasn't even thinking of where the period name would show up, that makes sense!

For putting the progress bar on the top circle, I would generally prefer it to be consistent, but I guess maybe we could justify it by saying the periods with non-numeric names are more important and therefore we want to display the progress bar more prominently?